src/mlpack/methods/elm/elm.hpp

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file elm.hpp
+ * @author Siddharth Agrawal
+ *
+ * Basic Extreme Learning Machine
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_ELM_HPP
+#define MLPACK_METHODS_ELM_HPP
+
+#include <mlpack/core.hpp>
+
+namespace mlpack {
+namespace elm {
+
+class ELM
+{
+private:
```

Comment:
Please make your style conform to requirements of mlpack. Like naming rule, how many space should be used etc.
You can check the details at [here](https://github.com/mlpack/mlpack/wiki/DesignGuidelines). 

examples : 

In mlpack, class member should names as
"trainTime", "weight" "NPrime" and so on.

space before private and public should be one.
space before parameter should be two